### ISSUE TITLE
The maxZoom and minZoom does not work if i set the values to "page-fit". It works fine with decimal values

### ISSUE THREAD

**Issue by Bharatvk** [stephanrauh/ngx-extended-pdf-viewer#2458]
I want to restrict the zoomlevels to page-fit so that users cannot change the zoom either by clicking the - or + buttons or by pinching. 

```html
<ngx-extended-pdf-viewer
              #pdfViewer
              [sidebarVisible]="isSidebarOpen"
              [base64Src]="pdfSource"
              pageViewMode="single"
              [page]="page"
              useBrowserLocale="true"
              backgroundColor="#f5f7fa"
              [showToolbar]="true"
              [textLayer]="false"
              [password]="pdfPassword"
              (pdfLoadingFailed)="pdfError($event)"
              [showBorders]="true"
              (pdfLoaded)="pdfLoaded($event)"
              (sidebarVisibleChange)="onSidebarVisibleChange($event)"
              [(zoom)]="zoomSetting"
              [maxZoom]="'page-fit'"
              [minZoom]="'page-fit'"
              [zoomLevels]="['page-fit']"
              [enablePinchOnMobile]="false"
              (currentZoomFactor)="onZoomChange($event)"
            ></ngx-extended-pdf-viewer>
```

I've tried on the demos in https://pdfviewer.net/extended-pdf-viewer/zoom as well, even here it doesn't seem to work

**Comment by stephanrauh** [comment]
It works the way I intended it to work - but you're right, this is a valid use-case. Comparing values like `page-fit` to `auto` or to a numerical value, so let's restrict this feature to one special case: `maxZoom === minZoom`.

**Comment by stephanrauh** [comment]
Your feature has landed with version 21.0.0-alpha.13.
